test/unit/lib/plugins/aws/package/lib/mergeIamTemplates.test.js

```diff
@@ -880,6 +263,14 @@ describe('lib/plugins/aws/package/lib/mergeIamTemplates.test.js', () => {
                 handler: 'index.handler',
                 disableLogs: true,
               },
+              fnWithVpc: {
+                handler: 'func.function.handler',
+                name: 'new-service-dev-func1',
```

Comment:
Do we need to configure this `name` property?

Reply:
We don't I removed it